ncm-network/src/main/perl/network.pm

```diff
@@ -1970,6 +2003,9 @@ sub Configure
     # main network config
     return if ! defined($self->mk_bu($NETWORKCFG));
 
+    # is NetworkManager is allowed?, false by default
```

Comment:
it's not `false` by default, because you have `defined(allow_nm)` in all the logic above.

Reply:
`allow_nm` was already part of the schema, one must explicitly define it as `true` in the profile otherwise it will be set to `false` by : `my $allow_nm = $nwtree->{allow_nm};`.

Reply:
OK, I'll remove them